official/resnet/cifar10_test.py

```diff
@@ -157,13 +168,27 @@ def test_cifar10model_shape_v2(self):
   def test_cifar10_end_to_end_synthetic_v1(self):
     integration.run_synthetic(
         main=cifar10_main.run_cifar, tmp_root=self.get_temp_dir(),
-        extra_flags=['-resnet_version', '1']
+        extra_flags=['-resnet_version', '1', '-batch_size', '4']
```

Comment:
how many steps do these tests run for?

Reply:
Just one. (It's one of the default args of `integration.run_synthetic`)